File: RDMPEG/RDMPEGPlayer/RDMPEGRawAudioFrame/RDMPEGRawAudioFrame.m
//
//  RDMPEGRawAudioFrame.m
//  RDMPEG
//
//  Created by Serhii Alpieiev on 9/7/17.
//  Copyright © 2017 Readdle. All rights reserved.
//

#import "RDMPEGRawAudioFrame.h"



NS_ASSUME_NONNULL_BEGIN

@interface RDMPEGRawAudioFrame ()

@property (nonatomic, strong) NSData *rawAudioData;

@end



@implementation RDMPEGRawAudioFrame

#pragma mark - Lifecycle

- (instancetype)initWithRawAudioData:(NSData *)rawAudioData {
    self = [super init];
    if (self) {
        self.rawAudioData = rawAudioData;
    }
    return self;
}

@end

NS_ASSUME_NONNULL_END

File: RDMPEG/RDMPEGPlayer/RDMPEGRawAudioFrame/RDMPEGRawAudioFrame.swift
//
//  RDMPEGRawAudioFrame.swift
//  RDMPEG
//
//  Created by Max Berezhnoy on 15/07/2024.
//  Copyright © 2024 Readdle. All rights reserved.
//

import Foundation

class RDMPEGRawAudioFrame {
    private(set) var rawAudioData: Data
    var rawAudioDataOffset: Int = 0

    init(rawAudioData: Data) {
        self.rawAudioData = rawAudioData
    }
}
